### DISCUSSION TITLE
material/table/

### DISCUSSION THREAD

**Discussion by giscus** [arungudelli/angularjswiki#174] [Announcements]
# material/table/

In this tutorial we are going to learn how to use Angular Material Table module in Angular with simple and detailed examples.

https://www.angularjswiki.com/material/table/

**Comment by WetieF** [comment]
Hi sir Could you please put the mat-pagination also in git. Thanks

**Reply by arungudelli** [reply]
Sure will do it

**Reply by arungudelli** [reply]
@WetieF I have added stackblitz demo link have a look
https://stackblitz.com/edit/angular-mat-table-pagination-example

**Comment by WetieF** [comment]
Thanks you so much Sir.

How can i have the entire Course of Angular from you Sir.

Best reagrds

Francis Liwat
________________________________

**Reply by arungudelli** [reply]
Will publish soon.. Thanks for asking..
